.gitignore

```diff
@@ -1,3 +1,6 @@
+# macOS
+.DS_Store
```

Comment:
This should go in your global gitignore rather than in each project. 

Not the guide I followed, but this was the first google result for a guide: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Want me to remove? A bunch of our other projects have this per-project but I'm fine either way.

Reply:
Please! We should be removing it anywhere it remains in favor of platform-global excludes.